doc/devel/contributing.rst

```diff
@@ -14,8 +14,60 @@ Submitting a bug report
 
 If you find a bug in the code or documentation, do not hesitate to submit a
 ticket to the
-`Bug Tracker <https://github.com/matplotlib/matplotlib/issues>`_. You are
-also welcome to post feature requests or pull requests.
+`Bug Tracker <https://github.com/matplotlib/matplotlib/issues>`_.
```

Comment:
I would add back the sentence "You are also welcome to post feature requests or pull requests" here, and remove the section down of details about the feature requests.